UAA: add known issue notes to affected patches

peterhaochen47 opened the pull request.

peterhaochen47 commented:
A similar note will need to be added to TAS 5 and 6 docs, but we'll hold off on making those PRs until this PR is reviewed. Thanks!

pspinrad commented:
LGTM, thanks-- will add to TAS 5 and 6 docs.

pspinrad commented:
Oh, wait @peterhaochen47 -- I don't know  which patch versions of TAS 5 and 6 this regression first showed up in.  Please LMK which patches or else PR to proper place-- thanks!

peterhaochen47 commented:
Hi @pspinrad, 
- for TAS 6: 6.0.0, 6.0.1 are affected, and fixed in 6.0.2
- for TAS 5: 5.0.10, 5.0.11 are affected, and fixed in 5.0.12 
Thank you!
